test/browserstack.sh

```diff
@@ -9,6 +9,7 @@ node_modules/.bin/testcafe 'browserstack:edge@17.0:Windows 10' test/testcafe/reg
 
 node_modules/.bin/testcafe 'browserstack:chrome:Windows 8.1' test/testcafe/regression-tests.js;
 node_modules/.bin/testcafe 'browserstack:firefox:Windows 8.1' test/testcafe/regression-tests.js;
+node_modules/.bin/testcafe 'browserstack:ie@11.0:Windows 8.1' test/testcafe/regression-tests.js;
```

Comment:
for context, ie11 was using the `shits-not-on-fire-tests.js` test script because the tracking lib wasn't sending any data because of no ie11 support for beacon, causing all the tests which counted sent events to fail

Reply:
ah, okay, good to know. i will fix ie or remove this test before merge